src/serialization/string_utils.cpp

```diff
@@ -865,4 +865,18 @@ void ellipsis_truncate(std::string& str, const std::size_t size)
 	}
 }
 
+std::string replace_all(const std::string& src, const std::string& from, const std::string& to)
```

Comment:
I think there's a `boost::replace_all` or similar which may already do exactly what you need.

Reply:
Will update.